README.rst

```diff
@@ -35,7 +35,7 @@ Allow user to mount (FreeBSD_)::
 
 ZFS_ Permissions::
 
-    zfs allow backup create,destroy,snapshot,rollback,clone,promote,rename,mount,send,receive,quota,reservation,hold storage
+    zfs allow <user> create,destroy,snapshot,rollback,clone,promote,rename,mount,send,receive,readonly,quota,reservation,hold <backup_dataset>
```

Comment:
Let's use SHELL variable notation for user and backup_dataset.  Example:

```
zfs allow ${USER} … ${BACKUP_DATASET}
```

What do you think?

Reply:
Good idea, changed